### PR TITLE
Fixes to packaging and various other minor changes

### PR DESCRIPTION
**Pull request by bwohlberg**
Fixes to packaging and various other minor changes.

### CODE REVIEW

File: docs/source/credits.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ We would like to thank the following sponsors for their financial support in the
 
 **Citation**
 
-Please use the following Bibtex citation when referencing this software. :cite:`sreehari2016plug` (coming soon) :cite:`venkatakrishnan2013plug`
+Please use the following bibtex citation when referencing this software. :cite:`sreehari2016plug` (coming soon) :cite:`venkatakrishnan2013plug`
```
Comment (bwohlberg) [inline docs/source/credits.rst:19]: I assume that this line was copied from a previous project, and should be deleted.

File: docs/source/credits.rst [hunk 2]
```diff
@@ -16,7 +16,7 @@ We would like to thank the following sponsors for their financial support in the
...
 ::
 
     @Misc {mbirjax-2024,
```
Comment (bwohlberg) [inline docs/source/credits.rst:22]: This reference seems to be a hybrid of a reference for a paper describing the new algorithm (title) and for the software itself (other entries). I assume the intent is to create two distinct entries. If this one is for the software, the title should match the software name.

Comment (cabouman) [reply docs/source/credits.rst:22]: Yes, this was just left over text. I moved it to the correct location and I removed the extra references from the bibliography. We don't yet have anything to cite on the VCD. I'm giving a poster at CVPR. Maybe I can cite that abstract once it's done.

**Review by cabouman — APPROVED** [review]
Brendt, I went over your changes and Greg's revisions, and they all look good to me. I also fixed up some errors I found in the documentation, and I fixed the citation errors you flagged. I'm going to accept the PR.

**Comment by cabouman** [comment]
Brendt, I did a squash merge, so I'm also deleting your branch.

**Pull request merged**